vllm/attention/ops/blocksparse_attention/interface.py

```diff
@@ -36,7 +35,7 @@ def __init__(
             use_spda = is_hip() or is_cpu() or not \
                        IS_COMPUTE_8_OR_ABOVE
         device = device or (torch.cuda.current_device()
-                            if torch.cuda.is_available() else "cpu")
+                            if current_platform.is_cuda_available() else "cpu")
```

Comment:
how is it different from `current_platform.is_cuda()` ?

Reply:
`is_cuda_available` is tied to `get_device_capability`. It currently returns `True` for cuda and rocm backend only.

Now that I think of it, `get_device_capability` probably initializes CUDA, so I'll update this to return a boolean directly based on the platform.

Reply:
can you add a new function `is_cuda_alike()`, that returns `True` for cuda and rocm? `is_available()` does not sound good to me, the current platform will always be available.